crates/typos-dict/assets/words.csv

```diff
@@ -48544,6 +48554,7 @@ thist,this
 thisy,this
 thiunk,think
 thius,this
+thje,the,their
```

Comment:
Not sure how someone would mistype "their" as "thje"

Reply:
Removed it at 7ee918f